devices/kurvia.js

```diff
@@ -0,0 +1,21 @@
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const extend = require('../lib/extend');
+const e = exposes.presets;
+
+const extendData = extend.light_onoff_brightness_colortemp_color({colorTempRange: [250, 454]});
+
+
+module.exports = [
+    {
+        zigbeeModel: ['ZB-CL01'],
+        model: 'ZB-CL01',
+        vendor: 'KURVIA',
+        description: 'GU10 GRBWC Built from Aliexpress',
+        exposes: extendData.exposes.concat(e.power_on_behavior()),
+        toZigbee: [tz.on_off].concat(extendData.toZigbee),
```

Comment:
Why is `tz.on_off` needed?

Reply:
Without this, the lamp does not turn off. Instead, it reduces the brightness to a minimum.

Reply:
Could you provide me the herdsman debug log when you turn off the device without the `tz.on_off`?

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

Reply:
Now I'm having trouble turning it on. Looks like I need to use cluster `genOnOff`  instead of `genLevelCtrl`

```
Debug 2022-07-23 01:00:51Received MQTT message on 'zigbee2mqtt/Corridor Spotlight/set' with data '{"state":"ON"}'
Debug 2022-07-23 01:00:51Publishing 'set' 'state' to 'Corridor Spotlight'
Debug 2022-07-23 01:00:51Received Zigbee message from 'Corridor Spotlight', type 'readResponse', cluster 'genLevelCtrl', data '{}' from endpoint 1 with groupID 0
Info 2022-07-23 01:00:51MQTT publish: topic 'zigbee2mqtt/Corridor Spotlight', payload '{"brightness":189,"color":{"x":0.4266,"y":0.4003},"color_mode":"color_temp","color_temp":317,"color_temp_startup":370,"linkquality":54,"power_on_behavior":"on","state":"ON"}'
Debug 2022-07-23 01:01:01Received MQTT message on 'zigbee2mqtt/Corridor Spotlight/get' with data '{"state":""}'
Debug 2022-07-23 01:01:01Publishing get 'get' 'state' to 'Corridor Spotlight'
Debug 2022-07-23 01:01:01Received Zigbee message from 'Corridor Spotlight', type 'readResponse', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
Info 2022-07-23 01:01:01MQTT publish: topic 'zigbee2mqtt/Corridor Spotlight', payload '{"brightness":189,"color":{"x":0.4266,"y":0.4003},"color_mode":"color_temp","color_temp":317,"color_temp_startup":370,"linkquality":54,"power_on_behavior":"on","state":"OFF"
```

Reply:
I see, thanks!